pkg/status/status_apiserver.go

```diff
@@ -30,3 +31,14 @@ func getLeaderElectionDetails() map[string]string {
 	leaderElectionStats["status"] = "Running"
 	return leaderElectionStats
 }
+
+func getDCAStatus() map[string]string {
+	clusterAgentDetails := make(map[string]string)
+	dcaCl, err := clusteragent.GetClusterAgentClient()
```

Comment:
The `init()` function of `DCAClient` only tries to create the endpoint from the environment variables and doesn't actually make a request to a `status` or `health` endpoint of the DCA. Is this enough to confirm that the DCA is working?

Reply:
Yes, that's what I was referring to in my comment - We can't reliably say if the DCA is working unless we add a health route in the DCA that will do some logic (specific to the metadata mapping).
That's why I changed it to `endpoint detected`.
We will add more insight in the dca check and when we add metrics.